src/middleware/error.middleware.ts

```diff
@@ -1,5 +1,8 @@
 // src/middleware/error.middleware.ts
 
+// a variable that contains specific information relevant to 400 Errors
+// it is structured using the http-exception.ts class
```

Comment:
Ahhhh,  please use the JSDoc standard when documenting code, this comment won't be picked up by intellisense. As per the previous comment, have a look at the following link on guidelines:
https://jsdoc.app/about-getting-started.html

Summarily, commenting/documenting code should explain why a specific function/class/etc exists, what its inputs are, and what the output generated is.